op-conductor/conductor/service.go

```diff
@@ -683,18 +687,27 @@ func (oc *OpConductor) transferLeader() error {
 }
 
 func (oc *OpConductor) stopSequencer() error {
-	oc.log.Info("stopping sequencer", "server", oc.cons.ServerID(), "leader", oc.leader.Load(), "healthy", oc.healthy.Load(), "active", oc.seqActive.Load())
-
-	_, err := oc.ctrl.StopSequencer(context.Background())
-	if err != nil {
-		if strings.Contains(err.Error(), driver.ErrSequencerAlreadyStopped.Error()) {
-			oc.log.Warn("sequencer already stopped.", "err", err)
+	oc.log.Info(
+		"stopping sequencer",
+		"server", oc.cons.ServerID(),
+		"leader", oc.leader.Load(),
+		"healthy", oc.healthy.Load(),
+		"active", oc.seqActive.Load())
+
+	// Getting stuck stopping a sequencer can't be good. Is it okay to fail to stop a sequencer on
+	// shutdown? From what I can tell it is.
+	latestHead, err := oc.ctrl.StopSequencer(oc.shutdownCtx)
```

Comment:
I think there's a race here to handle shutdown or stop the sequencer. Previously it could be asked to shutdown then it would try to stop the sequencer indefinitely.